docs/crash_reporting.md

```diff
@@ -62,6 +62,14 @@ In order to make your application's size smaller, its code is minified when it i
 -   dSYMs for your iOS native code
 -   Proguard mapping files if you have enabled code obfuscation for your Android native code
 
+To do setup your project for sending the symbolication files automatically run `npx @datadog/react-native-wizard`.
+
+See the wizard [official documentation][13] for options.
```

Comment:
```suggestion
```
This is an internal repo and there isn't much documentation there, so this link doesn't seem useful.

Reply:
@louiszawadzki do we plan on make the wizard open source?